sidebars.js

```diff
@@ -179,8 +179,11 @@ module.exports = {
           label: 'Automated Observability',
           collapsed: true,
           items: [
-            'platform-engineers/operations/observability',
```

Comment:
we should keep the path as it's a link in other docs such as blogs or docs.

Reply:
ok